dist/svg/ds4/icons.svg

```diff
@@ -159,13 +159,13 @@
         <path d="M8.316 21.161a.582.582 0 01-.583-.583c0-3.144.835-4.06 1.892-5.22l.074-.081c.622-.683 1.175-1.103 1.815-1.589a24.687 24.687 0 002.165-1.804c1.64-1.537 1.735-4.205 1.736-4.232 0-1.034-.17-1.972-.504-2.771a5.65 5.65 0 00-1.396-2.009 6.26 6.26 0 00-2.157-1.263c-.843-.294-1.797-.444-2.836-.444-1.142 0-2.186.199-3.1.59a6.529 6.529 0 00-2.313 1.642 7.495 7.495 0 00-1.46 2.515c-.342.969-.504 2.033-.482 3.162a.584.584 0 01-.57.595c-.309.027-.588-.249-.595-.57C-.023 7.83.16 6.628.55 5.527a8.65 8.65 0 011.69-2.905A7.669 7.669 0 014.961.685C6.022.231 7.221 0 8.521 0c1.17 0 2.254.172 3.222.511.98.344 1.84.849 2.556 1.501a6.793 6.793 0 011.686 2.42c.394.943.595 2.031.595 3.237-.003.144-.104 3.192-2.105 5.066a25.446 25.446 0 01-2.257 1.882c-.619.47-1.108.841-1.658 1.445l-.075.082c-.99 1.086-1.587 1.741-1.587 4.434a.584.584 0 01-.583.583h.001zM8.291 24a.582.582 0 01-.583-.583v-.742a.583.583 0 111.166 0v.742a.583.583 0 01-.583.583z"/>
     </symbol>
     <symbol id="icon-information" viewBox="0 0 24 24">
```

Comment:
Could of questions, why are we changing all the fills of info to #555? 
If we have a hover style, we should have no fill and have the CSS apply the fill color rather than changing the SVG fill color. 
I'm not sure I understand why this is changing and why only this one is switching (and not the others like attention and confirmation)